api/app/utils/redapp.py

```diff
@@ -318,6 +318,30 @@ def _fbp_fuel_type_map(fuel_type: str):
     20"S-2:  White Spruce / Balsam Slash"
     21"S-3:  Coastal Cedar / Hemlock / Douglas-Fir Slash"
     """
+    # fuel_type_map = {
```

Comment:
Commented out code

Reply:
Yeah - there's this weird lookup that's happening in REDapp, not sure we need it right now - but would like to keep the code here because it was a pain to copy.